KS.Fiks.Plan.JsonModelGenerator/Generate.cs

```diff
@@ -84,24 +79,24 @@ public static void Generate(string sourcePath, string outputFolder)
             {
                 Console.Out.WriteLine($"Typename: {codeArtifact.TypeName}");
                 
-                // Skip fellestyper og bruk using i stedet
-                if (fellesTyper.Contains(codeArtifact.TypeName))
+                // Legg på using hvis det er en fellestype
+                if (!schemaFilename.Contains(".felles.") && fellesTyper.Contains(codeArtifact.TypeName))
```

Comment:
Kan vi bruker const fellesSubNamespace her istedenfor å hardkode samme verdi på nytt?

Reply:
Enig 👍

Reply:
Hva er motivasjonen for å legge på denne ekstra sjekken?

if( !schemaFilename.Contains(".felles.") && ... )

Altså: i hvilken situasjoner er det ikke godt nok å kun sjekke om:

fellesTyper.Contains(codeArtifact.TypeName)

Reply:
Vi skal ikke legge på "using felles bla bla" når det er en felles fil. Den trenger ikke referer til seg selv